src/status_im/ui2/screens/chat/group_details/view.cljs

```diff
@@ -1,11 +1,9 @@
 (ns status-im.ui2.screens.chat.group-details.view
```

Comment:
just wondering why we still make changes in ui2 namespace, can we move to status-im2 ?

Reply:
Moved to status-im2